test/kickstart-templates/kickstart-centos.ks.template

```diff
@@ -2,7 +2,7 @@
 %include /main-network.cfg
 %include /main-cdrom.cfg
 
-%post --log=/var/log/anaconda/post-install.log --erroronfail
```

Comment:
@pacevedom wasn't there some issue with sos and this specific log file?

Reply:
There was trouble in CI because anaconda plugin was copying pull secrets. There was also trouble long ago when using embedded containers because anaconda files were huge. This change should be ok as logs go into the system console instead, so less dependency on anaconda.

Reply:
OK, this sounds safe then, thanks!